users/user-creation.md

```diff
@@ -0,0 +1,42 @@
+## User Creation
+
+Aside from [automatic account creation](/jetpack/automatic-account-creation-connection.md) there is an endpoint available for creating a WordPress.com user. This endpoint assumes the user creation is part of a product provisioning flow.
+
+
+### Endpoint Information (/user)
+
+- __Method__: POST
+- __URL__:    `https://public-api.wordpress.com/rest/v1.3/jpphp/user`
+
+### Request Parameters (/user)
+
+- __email__: The email address for the user that will be created.
+- __product_id__: The product id that is being provisioned for the user being created.
+
+
+### Response Parameters (/user)
+
+Below, the responses are grouped by whether the call to create a user was succesful or not.
+
+#### Successful response
+
+- __success__:       (bool) Was the operation successful?.
+- __bearer_token__: (string) The access token for the newly created user.
+
+#### Errored response
+
+- __error_code__:    (string) Error code, if any.
+- __error_message__: (string) Error message, if any.
+
+### Endpoint Errors (/user)
+
+The following is non-exhaustive list of errors that could be returned.
+
+| HTTP Code | Error Identifier          | Error Message                                                             |
+| --------- | ------------------------- | ------------------------------------------------------------------------- |
+| 403       | forbidden           | {partner} is not whitelisted for allowing user creation.                             |
```

Comment:
I don't think we need to align the pipes for these markdown tables. Not aligning them now might make future updates on ourselves easier as well.